src/alfasim_sdk/_internal/hook_specs.py

```diff
@@ -787,7 +782,7 @@ def finalize_state_variables_calculator(ctx: "void*") -> "int":
         :linenos:
         :emphasize-lines: 1
 
-        HOOK_FINALIZE_STATE_VARIABLES_CALCULATOR(void* ctx)
+        HOOK_FINALIZE_STATE_VARIABLES_CALCULATOR(ctx)
```

Comment:
It is OK. I mean, when the `alfasim-sdk template` generates a plugin, it adds macros in the `hook_spec.h` in which it is supposed to use without the type specification, since it is already added in the macro.

I removed it because I realize that when plugin developers start to implement one of those hooks, the first thing they do is copy/paste from the documentation. Since it is wrong to use the macro with type specification it can cause a compile error.